source/MultiGrid/MultiGrid.js

```diff
@@ -366,6 +369,8 @@ export default class MultiGrid extends Component {
         left: 0,
         outline: 0,
         overflow: 'hidden',
+        overflowX: 'hidden',
+        overflowY: 'hidden',
```

Comment:
Roger that. We should go ahead and get rid of the short-hand form (`overflow`) then. It's totally redundant.